### DISCUSSION TITLE
Importing to all Commerce Product Types

### DISCUSSION THREAD

**Discussion by seb-jones** [engram-design/FeedMe#1318] [General]
Howdy Folks,

Is it possible to run an import against *all* Commerce Products, regardless of their Product Type?

The context is that we have numerous Product Types (10+) but want to update prices across the entire range of products from a single Google Sheet.

Without this we would have to set up a sheet for each Product Type and a feed for each Product Type, which would be quite inconvenient for this number of Types!

Many thanks

Seb

**Comment by pdaleramirez** [comment]
@seb-jones feed me commerce product only supports single product type on the latest version.
